### PR TITLE
Equalizer

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Equalizer "Training maquetado"

## Trello Card
https://trello.com/c/DVTjLJsL/19-css-resets

### CODE REVIEW

File: maquetado/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+$black: #000;
```
Comment (henryzarza) [inline maquetado/_colors.scss]: Remember order by alphabetically

File: maquetado/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
```
Comment (henryzarza) [inline maquetado/_variables.scss]: Order by alphabetically

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good job only a few corrections

**Review by henryzarza — APPROVED** [review]
👍

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**